docs/book/features/volume_expansion.md

```diff
@@ -1,120 +1,78 @@
 
-The following  RBAC rules need to be added to support volume expansion:
+### RBAC Rules
 
-```bash
+Add the `patch` privilege to `persistentvolumes` resource and `update`, `patch` privileges to `persistentvolumeclaims/status` resource.
+
+```yaml
 - apiGroups: [""]
     resources: ["persistentvolumes"]
-    # NOTE: added "patch"
     verbs: ["get", "list", "watch", "update", "create", "delete", "patch"]
 - apiGroups: [""]
-    # NOTE: added "persistentvolumeclaims/status"
     resources: ["persistentvolumeclaims/status"]
     verbs: ["update", "patch"]
 ```
 
-## StorageClass
+### StorageClass
 
-In the StorageClass, set allowVolumeExpansion to true.
+Create a new StorageClass or edit the existing StorageClass to set `allowVolumeExpansion` to true.
 
-```bash
+```yaml
 kind: StorageClass
 apiVersion: storage.k8s.io/v1
 metadata:
-  name: example-vanilla-block-sc
-  namespace: kube-system
-  annotations:
-    storageclass.kubernetes.io/is-default-class: "false"
+  name: example-block-sc
 provisioner: csi.vsphere.vmware.com
 allowVolumeExpansion: true
-parameters:
-  # DatastoreURL: "ds:///vmfs/volumes/vsan:52cdfa80721ff516-ea1e993113acfc77/" # Optional Parameter
-  # StoragePolicyName: "vSAN Default Storage Policy"  # Optional Parameter
 ```
 
 ## Expand PVC Example
 
-Create a PVC with a StorageClass that allows volume expansion.
-
-```bash
-apiVersion: v1
-kind: PersistentVolumeClaim
-metadata:
-  name: example-vanilla-block-pvc
-spec:
-  accessModes:
-  - ReadWriteOnce
-  resources:
-    requests:
-      storage: 1Gi
-  storageClassName: example-vanilla-block-sc
-```
-
-```bash
-kubectl apply -f example-pvc.yaml
-```
-
-Increase the size of the PVC. Note that PVC cannot be attached to a Pod as only offline volume expansion is supported.
+Prior to increasing the size of a PVC make sure that the PVC is bound and is not attached to a Pod as only offline volume expansion is supported.
 
-```bash
-apiVersion: v1
-kind: PersistentVolumeClaim
-metadata:
-  name: example-vanilla-block-pvc
-spec:
-  accessModes:
-  - ReadWriteOnce
-  resources:
-    requests:
-      storage: 2Gi
-  storageClassName: example-vanilla-block-sc
-```
-
-When the PVC is not used by any pod, modify the PVC configuration to trigger volume expansion.
+Patch the PVC to increase its size:
 
 ```bash
-kubectl apply -f example-pvc.yaml
+kubectl patch pvc example-block-pvc -p '{"spec": {"resources": {"requests": {"storage": "2Gi"}}}}'
 ```
 
-The Cloud Native Volume size in vCenter will be changed to the new size. Size of PV will also be changed with that.
-
-Note: The size of PVC won’t be changed until the PVC is used by a new Pod.
+This will trigger an expansion in the volume associated with the PVC in vSphere Cloud Native Storage which finally gets reflected on the size of the corresponding PV object. Note that the size of PVC will not change until the PVC is attached to a node i.e used by a Pod.
```

Comment:
Note that the size -> Note that the capcity

Reply:
Good catch, will change it!

Reply:
Done